script/RumpelConfig.sol

```diff
@@ -4075,6 +4091,43 @@ library RumpelConfig {
 
         return configs;
     }
+
+    function getHyperevmBeHypeProtocolConfigs() internal
+        pure
+        returns (ProtocolGuardConfig[] memory) {
+                    ProtocolGuardConfig[] memory configs = new ProtocolGuardConfig[](2);
+
+        configs[0] = ProtocolGuardConfig({
+            target: HYPEREVM_HYPERBEAT_VAULT_DEPOSIT_BEHYPE,
+            selectorStates: new SelectorState[](1)
+        });
+        configs[0].selectorStates[0] =
```

Comment:
interesting it has a different interface to the liquidhype depoist vault, but I think there's more I need to understand about these – looks correct to me 👍